app/controllers/application_controller.rb

```diff
@@ -44,7 +44,7 @@ def groups_from_request_env
       raw_header = roles if Rails.env.development?
       # This test setting is for cypress.
       raw_header = roles if Rails.env.test? && roles
-      logger.debug("Roles are #{raw_header}")
+      logger.info("Rails Env: #{Rails.env}; User: #{current_user}; Roles: #{raw_header}")
       raw_header&.split(";") || []
```

Comment:
Note: since we do something different in non-production Rails env, let's just log that too for sanity.  Also, log the username so we can be sure we are dealing with the user we expect.  

Since this is only run once per session for a given user, it will only happen when they first log in and get a new session (i.e. it will not be noisy in the logs).